app/services/podcasts/feed.rb

```diff
@@ -26,7 +26,7 @@ def get_episodes(limit: 100, force_update: false)
       # https://github.com/forem/forem/issues/3580, in which some RSS feeds list their episodes in
       # earliest to latest, whereas others list latest to earliest.  We assume that we're wanting to
       # track the latest.
-      feed.items.sort { |a, b| b.pubDate <=> a.pubDate }.first(limit).each do |item|
+      feed.items.sort { |a, b| b.pubDate.to_i <=> a.pubDate.to_i }.first(limit).each do |item|
```

Comment:
@jeremyf I would like to gut check with you if this is an acceptable approach to pretend nil pubDate as 0.

Reply:
I'm alright with this.

The next thing would be to fall back to index position (like first sort on pubDate then sort on index position).  But that's not available.

Reply:
I confirmed that `RSS::Parser.parse` turns `pubDate` into actual `Time` objects, so this should be fine. BTW, in earlier Ruby days I often saw this idiom for descending sorts. It would work here but it's maybe a bit obscure?

```ruby
feed.items.sort_by { |i| -i.pubDate.to_i }
```

Reply:
If there's a code comment, go with that idiom!